chapters/en/slides/module2/module2_03.Rmd

```diff
@@ -0,0 +1,284 @@
+---
+params:
+  dynamictitle: "module2_03"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+# Creating bar charts and histograms
+
+Notes:
+In this module
+we will be learning how to create bars charts for counting observations,
+and a special types of bar chars called histograms
+for showing distributions of data.
+
+---
+
+## Subsetting the global development data
+
+```{python}
+import pandas as pd
+
+
+url = 'https://raw.githubusercontent.com/UofTCoders/workshops-dc-py/master/data/processed/world-data-gapminder.csv'
+gm = pd.read_csv(url, parse_dates=['year'])
+gm2018 = gm[gm['year'] == '2018']
+gm2018
+```
+
+Notes:
+In this slide deck,
+we will continue to work with the gapminder dataset.
+To make it more suitable for this slide deck
+let's subset it to include observations from the year 2018 only.
+
+When filtering on a date in pandas,
+we can either type out just the year portion
+or the entire date string in the format shown in the slide.
+
+---
+
+## Creating a bar chart of the total population per continent
+
+```{python}
+import altair as alt
+
+
+alt.Chart(gm2018).mark_bar().encode(
+    x='region',
+    y='sum(population)')
+```
+
+Notes:
+Bar charts are often avoided when showing summary statistics such as the mean
+because they hide any variation,
+something we will talk more about in later modules.
+
+But they are fine for showing statistics
+such as counts of observations, summations, or individual values
+(e.g. the number of people in a country for a specific year).
```

Comment:
We need to fill in the because here, but my brain is failing me at the moment...

```suggestion
(e.g. the number of people in a country for a specific year) because ....
```

Reply:
Suggestions from Tom and Alexi:

Tom: something like: ...because such statistics represent a single (unsummarized/ungrouped) piece of information?

Alexi: "... and also they are not suitable for continuous statistics. There are another alternatives for this class of statistics such as boxplots, violin plots, or histograms."

Reply:
Changed to:

Bar charts are often avoided when visualizing summary statistics
where it is important to display the variation in the raw data.

This is the case for measures of central tendency,
such as the mean and median,
which are better represented with other visualizations
that we will cover later in the course.

Bar chart are a good choice for representing individual values,
such as the number of people living in a country
at a specific point in time.

Some summary statistics are displayed as single values,
such as counts of observations and summations,
and bar charts are suitable for these as well.

We will talk more about how to best represent summary statistics 
and variation in raw data when we learn about distributions
in the next module.